src/client/conn/http1.rs

```diff
@@ -302,6 +302,12 @@ where
 
 // ===== impl Builder
 
+impl Default for Builder {
```

Comment:
Since this _adds_ something new, let's leave it as a feature PR to be separate.

Reply:
OK.